script.js

```diff
@@ -1,98 +1,152 @@
 const gridContainer = document.querySelector(".grid-container");
 
-// use populateCells function to do what the name suggests 
-populateCells(16*16); 
-
+// use populateCells function to do what the name suggests
+populateCells(16 * 16);
 
 // Add event listener to the create button
-document.querySelector("#create").addEventListener("click",function(){
-    createGrid();
-});   
+document.querySelector("#create").addEventListener("click", function () {
+  createGrid();
+});
+
+// Add event listener to the reset button
+document.querySelector("#reset").addEventListener("click", function () {
+  const response = confirm("Are you sure?? Everything will be deleted");
+  if (response) {
+    body.style.setProperty("--rows", 16);
+    body.style.setProperty("--cols", 16);
+    populateCells(16 * 16);
+  }
+});
+
+// Add event listener to the remove grid button
+document.querySelector("#remove-grid").addEventListener("click", function () {
+  removeGrid();
+});
+
+// Add event listener to the clear button
+document.querySelector("#clear").addEventListener("click", function () {
+  const response = confirm("Are you sure?? Everything will be deleted");
+  if (response) clear();
+});
 
 let isSketching = false;
 let isErasing = false;
-// Add event listener (keydown) to the document to capture (s) and (e) keys in addtion to 
+// Add event listener (keydown) to the document to capture (s) and (e) keys in addtion to
 // (Enter) key
-document.addEventListener("keydown",function(event){
-    if(event.code ==="KeyS"){
-        isSketching = !isSketching;
-        isErasing = false;
-    }
-    if(event.code === "Enter"){
-        createGrid();
-        document.querySelector("#rows").blur(); 
-        document.querySelector("#cols").blur();
-    }
-    if(event.code === "KeyE"){
-       isErasing = !isErasing; 
-       isSketching = false;
-    }
-}); 
-
+document.addEventListener("keydown", function (event) {
+  if (event.code === "KeyS") {
+    isSketching = !isSketching;
+    isErasing = false;
+  }
+  if (event.code === "Enter") {
+    createGrid();
+    document.querySelector("#rows").blur();
+    document.querySelector("#cols").blur();
+  }
+  if (event.code === "KeyE") {
+    isErasing = !isErasing;
+    isSketching = false;
+  }
+});
 
 // Mouse Over Event
 // Add event listener to the grid container (this will include all the cells inside it)
-gridContainer.addEventListener("mouseover",function(event){
-    if(isSketching){
-        let id = event.target.id;
-        let cellBackgroundColor = getComputedStyle(event.target).getPropertyValue("background-color"); 
-        if(cellBackgroundColor === "rgb(255, 255, 255)"){
-            let randColor = randomColor(); 
-            event.target.style.backgroundColor = convertRGBtoHEX(randColor);
-        }
-    }if(isErasing){
-        erase(event);
+gridContainer.addEventListener("mouseover", function (event) {
+  if (isSketching) {
+    let id = event.target.id;
```

Comment:
This variable declared here but never used.

Reply:
It will be used soon.